src/securityPolicy.c

```diff
@@ -296,7 +296,7 @@ security_policy_t policyForSignTxInput()
 // For each transaction (third-party) address output
 security_policy_t policyForSignTxOutputAddressBytes(
         sign_tx_signingmode_t txSigningMode,
-        const uint8_t* rawAddressBuffer, size_t rawAddressSize,
```

Comment:
I think we use size_t everywhere for buffer size, so I'd keep it here, too.

Reply:
Done.